mappings/net/minecraft/world/VoxelView.mapping

```diff
@@ -35,6 +38,10 @@ CLASS net/minecraft/class_4538
 		ARG 1 pos
 	METHOD method_22351 isWater (Lnet/minecraft/class_2338;)Z
 		ARG 1 pos
+	METHOD method_22387 accessGeneratorBiome (III)Lnet/minecraft/class_1959;
```

Comment:
hm, not sure about the `access` prefix here. maybe something like `getStoredBiome`?

Reply:
this is like if the world doesn't have a chunk loaded, it will "access" (call that method in BiomeAccessor.Storage) the biome stored in generator